tools/mo/openvino/tools/mo/front/onnx/group_norm_ext.py

```diff
@@ -15,8 +16,8 @@ class ExperimentalDetectronGroupNorm(FrontExtractorOp):
     @classmethod
     def extract(cls, node):
         attrs = {
-            'eps': np.array(onnx_attr(node, 'eps', 'f', default=1e-6), dtype=np.float),
-            'num_groups': np.array(onnx_attr(node, 'num_groups', 'i', default=1), dtype=np.int64),
+            'eps': mo_array(onnx_attr(node, 'eps', 'f', default=1e-6), dtype=np.float),
```

Comment:
float32_array?

Reply:
It is eps, it is float64 array by design